blitz/api/blitz_api.py

```diff
@@ -128,6 +131,7 @@ def create_blitz_api(
 ) -> BlitzAPI:
     from blitz.ui.main import init_ui
 
+    blitz_app_version = None
```

Comment:
I added this line because in case of --no-hot-reload option on start command, the startup crashed with:
`UnboundLocalError: cannot access local variable 'blitz_app_version' where it is not associated with a value`